src/Microsoft.SqlTools.ServiceLayer/Utility/CommandOptions.cs

```diff
@@ -82,10 +102,32 @@ public string Usage
                     "Microsoft.SqlTools.ServiceLayer.exe " + Environment.NewLine +
                     "   Options:" + Environment.NewLine +
                     "        [--enable-logging]" + Environment.NewLine +
-                    "        [--help]" + Environment.NewLine,
+                    "        [--help]" + Environment.NewLine +
+                    "        [--locale]" + Environment.NewLine,
```

Comment:
Right now I just have is print an exception telling you the locale is not supported to console and fall back to en-US.